NF2FF Directivity calculation seems wrong when radius != 1

gammaxy1 opened the pull request:
When I set radius>1 when calling CalcNF2FF() from Python, Dmax ends up being lower than expected. I expected Dmax to be independent of radius.

This fix includes a radius^2 factor in the calculation of m_maxDir.

This radius^2 factor was once included in the P_max calculation but was removed because it didn't belong there. I think putting it in m_maxDir will complete the fix.

thliebig commented:
Well I think you fix is correct after all... P_max is W/m² not W per unit solid angle... I will have a closer look once more.

gammaxy1 commented:
I added some simple comments and changed the ordering of the math for clarity. Also fixed my name and email address.

Thanks for all you do with this great project.
